.circleci/config.yml

```diff
@@ -6,37 +6,31 @@ build_allways: &build_allways
     tags:
       only: /.*/
 defaults: &defaults
-  working_directory: /home/circleci/.go_workspace/src/github.com/kubeless/kubeless
+  working_directory: /home/circleci/.go_workspace/src/github.com/kubeless/kafka-trigger
   environment:
     GOPATH: /home/circleci/.go_workspace
     CONTROLLER_IMAGE_NAME: bitnami/kubeless-controller-manager
-    BUILDER_IMAGE_NAME: kubeless/function-image-builder
     KAFKA_CONTROLLER_IMAGE_NAME: bitnami/kafka-trigger-controller
-    NATS_CONTROLLER_IMAGE_NAME: bitnami/nats-trigger-controller
-    KINESIS_CONTROLLER_IMAGE_NAME: bitnami/kinesis-trigger-controller
     CGO_ENABLED: "0"
     TEST_DEBUG: "1"
     GKE_VERSION: 1.8.8-gke.0
     MINIKUBE_VERSION: v0.25.2
-    MANIFESTS: kubeless kubeless-non-rbac kubeless-openshift kafka-zookeeper kafka-zookeeper-openshift nats kinesis
+    MANIFESTS: kafka-zookeeper kafka-zookeeper-openshift
```

Comment:
we probably want to use here a certain version of kubeless

Reply:
Right now CI uses latest _released_ version of Kubeless function controller, and not `latest` image. I dont foresee a situation where Kafka trigger controller can not be compatible with latest release of Kubeless. Just wanted to avoid hard coding to a particular version right now. If need arises we can do so.